scripts/run_docker

```diff
@@ -11,6 +12,9 @@ else
   SSL_PATH=ssl
 fi
 export HOST
+
+export GIT_SHA=$(git -C . rev-parse --short=7 HEAD)
+
 docker-compose up -d
 docker cp app/server/src/resources/config.json beebop_beebop-server_1:/app/src/resources/config.json
 docker cp proxy/ssl/dhparam.pem beebop_proxy_1:/run/proxy/
```

Comment:
This failed for me; docker-compose has brought things up as `beebop-proxy-1`, etc.

I have

```
rfitzjoh@wpia-dide300:~/Documents/Projects/bacpop/beebop$ docker-compose --version
Docker Compose version v2.24.1
```

It looks like we can pass a flag in here to get this behaviour back: https://docs.docker.com/compose/migrate/#service-container-names

or we should probably update to use a newer docker-compose everwhere